vars/karmaExecuteTests.groovy

```diff
@@ -0,0 +1,73 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.GitUtils
+import com.sap.piper.Utils
+
+import groovy.text.SimpleTemplateEngine
+import groovy.transform.Field
+
+@Field String STEP_NAME = 'karmaExecuteTests'
+@Field Set GENERAL_CONFIG_KEYS = [
+    'containerPortMappings', //port mappings required for containers. This will only take effect inside a Kubernetes pod, format [[containerPort: 1111, hostPort: 1111]]
```

Comment:
javadoc syntax `/** */` instead of `//` fits better with generating the docu out of the step here.

Reply:
Haven't had a look at the docs generation yet, but will change it.